packages/i18n/src/components/richTextElements.tsx

```diff
@@ -66,7 +66,9 @@ const secondary = (text: ReactNode) => (
  * @param text  text to wrap
  */
 const red = (text: ReactNode) => (
-  <span data-h2-color="base(error.dark)">{text}</span>
+  <span data-h2-color="base(error.darker) base:dark(error.lightest)">
```

Comment:
Does this change work for colour contrast for the other instance where this is used?

https://github.com/GCTC-NTGC/gc-digital-talent/blob/6828bd863471c118ed3c9e75e877fad15c0966b8/apps/web/src/pages/DirectiveForms/DigitalServicesContractingQuestionnaire/sections/InstructionsSection.tsx#L37

Reply:
From 3.75 to 8.40 for dark mode view